examples/pysdl2.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python2
+
+#
+# Simple SDL2 / cefpython3 example.
+#
+# Only handles mouse events but could be extended to handle others.
+#
+# Requires pysdl2 (and SDL2 library).
```

Comment:
pip2 install PySDL2

Note: you will need to make sure you have installed the SDL2 libraries for your OS.